dubbo-rpc/dubbo-rpc-triple/src/test/java/org/apache/dubbo/rpc/protocol/tri/TripleProtocolTest.java

```diff
@@ -45,6 +43,19 @@
 
 class TripleProtocolTest {
 
+    @Test
+    void testPbMethodOverride()throws Exception{
```

Comment:
The test should not be here, I think the test constructor ReflectionServiceDescriptor is better

Reply:
> The test should not be here, I think the test constructor ReflectionServiceDescriptor is better

The project which contains ReflectionServiceDescriptor.java code did't have protobuf dependencies,So I cannot add pb class for this unit test.Can i add protobuf denpendices to pom.xml.

Reply:
`scope=test` can solve this problem

Reply:
i fix it